datatypes/blueprints/datatypes.py

```diff
@@ -7,8 +7,8 @@
 from sqapi.api import responding
 
 SELECT_ALL_DATA_TYPES = 'select_datatypes_count.sql'
-SELECT_UUIDREFS_BY_DATA_TYPES = 'select_uuidrefs_by_datatypes.sql'
-SELECT_UUIDREFS_BY_DATA_TYPES_ON_DATE = 'select_uuidrefs_by_dt_on_date.sql'
+SELECT_UUIDREFS_BY_DATA_TYPES = 'select_by_datatype.sql'
```

Comment:
dersom du har renamet UUID_REF -> UUID, kan du vel også kalle disse for SELECT_UUID_BY_*